### PR TITLE
refactor(api): move getSingleItems to separate files

### PR DESCRIPTION
**Pull request by lionelB**

### CODE REVIEW

File: packages/code-du-travail-api/src/server/routes/items/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  const relatedItems =
+    ctx.params.source === "faq" ? await searchItemsFromTheme(item) : {};
+
+  ctx.body = {
+    ...item,
+    relatedItems
+  };
+});
+
+router.get(`${routeName}/:id`, async ctx => {
```
Comment (revolunet) [inline packages/code-du-travail-api/src/server/routes/items/index.js:52]: maybe add some jsdoc. is this stille used ?

Comment (lionelB) [reply packages/code-du-travail-api/src/server/routes/items/index.js:52]: not used for now, but could be one possible fix for our uniq slug problem (move from slug to id in url)

**Review by revolunet — APPROVED** [review]

File: packages/code-du-travail-api/src/server/routes/items/index.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+  if (results.aggregations.bySource.buckets.length === 0) {
+    return {};
+  }
+
+  const { buckets } = results.aggregations.bySource;
+
+  return buckets.reduce((state, bucket) => {
+    if (bucket.doc_count > 0) {
+      state[bucket.key] = bucket.bySource.hits.hits;
```
Comment (douglasduteil) [inline packages/code-du-travail-api/src/server/routes/items/index.js]: could be :

```js
return bucket.doc_count > 0 ? { ...state, [bucket.key]: bucket.bySource.hits.hits } : state
```

:heart_eyes:

**Review by douglasduteil — APPROVED** [review]
C'est de toute beauté :heart:

**Pull request merged**